lib/ProductOpener/Users.pm

```diff
@@ -367,19 +369,26 @@ sub init_user()
 {
 	my $user_id = undef ;
 	my $user_ref = undef;
+	my $org_ref = undef;
+
 	my $cookie_name = 'session';
+	my $cookie_domain = "." . $server_domain;	# e.g. fr.openfoodfacts.org sets the domain to .openfoodfacts.org
+	if (defined $server_options{cookie_domain}) {
+		$cookie_domain = "." . $server_options{cookie_domain};	# e.g. fr.import.openfoodfacts.org sets domain to .openfoodfacts.org
+	}
 
 	$cookie = undef;
 
 	$Visitor_id = undef;
 	$Facebook_id = undef;
 	$User_id = undef;
+	$Org_id = undef;
```

Comment:
How/when is this initialized?

Reply:
Right now it is a placeholder, we'll need to add forms options etc. to create orgs.